templates/trick/index.html.twig

```diff
@@ -0,0 +1,49 @@
+{%  extends 'base.html.twig' %}
+
+{% block title %}
+    Liste des figures
+{% endblock %}
+
+{% block body%}
+        <div class="container mb-5 mt-4 " id="section">
+            <div class="bg-light ">
+                <div class="row">
+                    <div class="col-12 col-md-8"><h1 class="p-2">Les figures</h1></div>
+                    <div class=" col-12 col-md-4 d-flex align-items-center justify-content-end "><a href="#" type="button" class="btn btn-sm btn-outline-secondary mr-3">Ajouter une figure</a></div>
+                </div>
+
+                <hr>
+                {% for message in app.flashes('success') %}
+                    <div class="alert alert-success text-center" role="alert">
+                        {{ message }}
+                    </div>
+                {% endfor %}
+
+                <div class="row p-2">
+                    {% for  trick in tricks %}
+                        <div class="col-12  col-lg-3">
+                            <div class="card mb-4 shadow-sm">
+                                {% for media in medias %}
```

Comment:
Du coup au lieu de faire `{% for media in medias %}` tu peux faire directement `{% for media in trick.medias %}`

Reply:
ok modification faite